.gitignore

```diff
@@ -28,3 +28,6 @@ wasm/*.wasm
 
 # app version string file
 /apps/version.rs
+
+# mac
+.DS_Store
```

Comment:
pls don't add OS specific settings here - you can add it to your global git config

Reply:
Removed. 

Thank you!